components/global_header/left_controls/product_menu/product_menu_list/product_menu_list.tsx

```diff
@@ -103,6 +120,22 @@ const ProductMenuList = (props: Props): JSX.Element | null => {
                         />
                     }
                 />
+                <Menu.ItemToggleModalRedux
```

Comment:
I guess that is about UX/QA, but is this always visible? to anybody? there is no permission needed? or license? or config?